src/components/Event.jsx

```diff
@@ -179,6 +198,7 @@ const Event = ({
         </PlayerWrapper>
       </DummyContainer>
       <PlayerContainer isFixed={isFixed}>
+        <PlayerCloseButton onClick={onPlayerCloseButtonClick} isFixed={isFixed}>x</PlayerCloseButton>
```

Comment:
I would change this to use semantic-ui-react Button and Icon components:
 ```suggestion
 <PlayerCloseButton icon onClick={onPlayerCloseButtonClick} isFixed={isFixed}>
   <Icon inverted name='close' size='large' />
 </PlayerCloseButton>
```